i18n/it/campaigns/eoe/campaign.po

```diff
@@ -11,308 +11,367 @@ msgstr ""
+msgstr "Tekeli-li"
 
 msgid ""
-"“That seething, half-luminous cloud-background held ineffable suggestions of a vague, ethereal <i>beyondness</i> far more than terrestrially spatial; and gave appalling reminders of the utter remoteness, separateness, desolation, and aeon-long death of this untrodden and unfathomed austral world.”\n"
-"<i>–H. P. Lovecraft,</i> At the Mountains of Madness"
+"This campaign includes an encounter set of 16 weaknesses, each with the title \"Tekeli-li!\" These cards are shuffled together to form a special Tekeli-li deck during the setup of each scenario of this campaign.\n"
+"Like normal weaknesses, if one of these cards is added to an investigator's deck, it becomes a part of that deck and stays with that investigator from scenario to scenario. However, the resolution of each Tekeli-li! weakness instructs the investigator to return it to the Tekeli-li deck (removing it from their own deck). As a result, the number of cards in the Tekeli-li deck varies based on how many of these weaknesses are in each investigator's deck.\n"
+"<i>Note: While all Tekeli-li! cards share the same title, they may have differing effects.</i>"
 msgstr ""
+"Questa campagna include un set di incontri di 16 debolezze, ognuna con il nome \"Tekeli-li\". Queste carte vengono mescolate per comporre uno speciale mazzo Tekeli-li durante la preparazione di ogni scenario di questa campagna.\n"
+"Come per le normali debolezze, se una di queste carte viene aggiunta al mazzo di un investigatore, diventa parte di quel mazzo e resta con quell'investigatore tra uno scenario e l'altro. Tuttavia, la risoluzione di ogni debolezza Tekeli-li richiede all'investigatore di rimetterlo nel mazzo Tekeli-li (rimuovendola dal suo mazzo). Di conseguenza, il numero di carte nel mazzo Tekeli-li varierà in base a quante di queste debolezze sono nei mazzo degli investigatori.\n"
+"<i>Nota: Sebbene le carte Tekeli-li abbiano tutte lo stesso nome, possono avere effetti differenti.</i>"
 
-msgid "<i>I want to return from the expedition</i>"
-msgstr ""
+msgid "Partner"
+msgstr "Compagno"
 
-msgid "<i>I want to struggle against the elements</i>"
-msgstr ""
+msgid "Partner is a keyword ability that appears on the nine story assets in the <i>Expedition Team</i> encounter set. Each of these assets represents a powerful ally whom investigators can bring along during scenarios in this campaign to improve their chances of success. However, bringing an asset with the partner keyword endangers that asset, with the risk of losing them permanently."
+msgstr "Compagno è una capacità di parola chiave che compare sui 9 supporti storia nel set di incontri <i>Squadra della Spedizione</i>. Ognuno di questi supporti rappresenta un potente alleato che gli investigatori possono portare con sé durante gli scenari di questa campagna per aumentarne le possibilità di successo. Tuttavia, portare un supporto con Compagno metterà in pericolo quel supporto, con il rischio di perderlo definitivamente"."
```

Comment:
The quotation mark at the end of this entry needs to be escaped `definitivamente\"."` or replaced with a smart quote

Reply:
Ops, sorry. Solved!